From 684bae6043fa2242a5c6551c3f260c9db0cf7dc8 Mon Sep 17 00:00:00 2001
From: Andreas Klinger <ak@it-klinger.de>
Date: Mon, 4 Jan 2021 08:40:50 +0100
Subject: [PATCH] configure.ac: fix error with version string

When autoreconfiguring there's an error: "configure.ac:6: error: AC_INIT
should be called with package and version arguments".

Put version string in square brackets to satisfy autoconfigure.

Signed-off-by: Andreas Klinger <ak@it-klinger.de>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 65387d6..7a0054e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_PREREQ(2.61)
-AC_INIT(ply, m4_esyscmd_s(git describe --always --dirty),
+AC_INIT(ply, [2.1.1],
 	     https://github.com/wkz/ply/issues)
 
 AC_GNU_SOURCE
-- 
2.20.1

